trogon/widgets/command_info.py

```diff
@@ -43,7 +43,7 @@ def on_mount(self) -> None:
                     getattr(schema.parent, "name", "No parent"),
                 ),
                 (Text("Subcommands", style="b"), list(schema.subcommands.keys())),
-                (Text("Group", style="b"), schema.is_group),
+                (Text("Group", style="b"), bool(schema.subcommands)),
```

Comment:
A command is a group if it has subcommands; i.e.:

`is_group = bool(schema.subcommands)`